src/providers/bttv/BttvEmotes.cpp

```diff
@@ -218,18 +222,35 @@ void BttvEmotes::loadEmotes()
         return;
     }
 
-    NetworkRequest(QString(globalEmoteApiUrl))
-        .timeout(30000)
-        .onSuccess([this](auto result) {
-            auto emotes = this->global_.get();
-            auto pair = parseGlobalEmotes(result.parseJsonArray(), *emotes);
-            if (pair.first)
-            {
-                this->setEmotes(
-                    std::make_shared<EmoteMap>(std::move(pair.second)));
-            }
-        })
-        .execute();
+    readProviderEmotesCache("global", "bttv", [this](auto jsonDoc) {
+        auto emotes = this->global_.get();
+        auto pair = parseGlobalEmotes(jsonDoc.array(), *emotes);
+        if (pair.first)
+        {
+            this->setEmotes(std::make_shared<EmoteMap>(std::move(pair.second)));
+        }
+    });
+
+    std::ignore = QtConcurrent::run([this]() {
```

Comment:
This goes for all of these uses: We don't need to initiate this in a different thread. The network request will already run in a different thread. You should've seen assertions fail if you built in debug mode.